### ISSUE TITLE
Wire.h generating error messages. Seems to affect my project hardware.

### ISSUE THREAD

**Issue by kee** [sparkfun/SparkFun_Qwiic_Relay_Arduino_Library#7]
**TLDR**

"Verify" in the Arduino IDE generates the error messages shown below. The code I've written seems to work when the relay is OFF but when the relay turns ON, serial LCD display looks like it is power cycling. Did not do that in April.

**Version Details**

Am using Arduino Uno (Sparkfun RedBoard actually) with a COM-15093 SparkFun Qwiic Single Relay that uses the SparkFun Qwiic Relay Arduino Library version 1.2.0.
Have a 1 amp 12 volt DC power supply plugged into the RedBoard so everything should have sufficient power. (With the relay OFF everything works as designed and the serial display stays on and updates the display with power just coming from the USB cable.)
Last time I verified this code was April 2023. I switched from an Intel Mac to an M2 Mac and put this aside for months to focus on other stuff. It's now September 2023 and I'm returning to this code. Updated the Arduino IDE to 2.2.1. Updated all the other libraries I had installed in April 2023. Re-uploaded the code into the Uno. Turns out none of the SparkFun accessory board libraries have changed.
When I do "Verify" in the IDE I get these error messages and I'm not sure what they mean, why I'm getting them, and whether they are causing the problems (I assume so since changing the relay from OFF to ON causes the problems.)

**Hardware**

Everything is connected via Qwiic cables from SparkFun. This is the order in which they are connected:
DEV-15123 SparkFun RedBoard Qwiic
DEV-15164 SparkFun Qwiic OpenLog
SEN-16467 SparkFun Humidity Sensor Breakout - SHTC3 (Qwiic)
COM-15093 SparkFun Qwiic Single Relay
BOB-16842 SparkFun Qwiic Button - Green LED - address: 0x51
BOB-16842 SparkFun Qwiic Button - Green LED - address: 0x52
LCD-16398 SparkFun 20x4 SerLCD - RGB Backlight (Qwiic)

**Error Messages**

In file included from /Users/keenethery/Documents/Arduino/libraries/SparkFun_Qwiic_Relay_Arduino_Library/src/SparkFun_Qwiic_Relay.h:4:0,
                 from /Users/keenethery/Documents/Arduino/libraries/SparkFun_Qwiic_Relay_Arduino_Library/src/SparkFun_Qwiic_Relay.cpp:12:
/Users/keenethery/Library/Arduino15/packages/arduino/hardware/avr/1.8.6/libraries/Wire/src/Wire.h: In member function 'uint8_t Qwiic_Relay::_readCommand(uint8_t)':
/Users/keenethery/Library/Arduino15/packages/arduino/hardware/avr/1.8.6/libraries/Wire/src/Wire.h:68:13: note: candidate 1: uint8_t TwoWire::requestFrom(int, int)
     uint8_t requestFrom(int, int);
             ^~~~~~~~~~~
/Users/keenethery/Library/Arduino15/packages/arduino/hardware/avr/1.8.6/libraries/Wire/src/Wire.h:65:13: note: candidate 2: uint8_t TwoWire::requestFrom(uint8_t, uint8_t)
     uint8_t requestFrom(uint8_t, uint8_t);
             ^~~~~~~~~~~
/Users/keenethery/Library/Arduino15/packages/arduino/hardware/avr/1.8.6/libraries/Wire/src/Wire.h: In member function 'float Qwiic_Relay::_readVersion(uint8_t)':
/Users/keenethery/Library/Arduino15/packages/arduino/hardware/avr/1.8.6/libraries/Wire/src/Wire.h:68:13: note: candidate 1: uint8_t TwoWire::requestFrom(int, int)
     uint8_t requestFrom(int, int);
             ^~~~~~~~~~~
/Users/keenethery/Library/Arduino15/packages/arduino/hardware/avr/1.8.6/libraries/Wire/src/Wire.h:65:13: note: candidate 2: uint8_t TwoWire::requestFrom(uint8_t, uint8_t)
     uint8_t requestFrom(uint8_t, uint8_t);
             ^~~~~~~~~~~

**Help Please**

Anyone have any thoughts on if fixing the errors in Wire.h will fix the power cycling of the Serial LCD display?

**Comment by edspark** [comment]
Yes, I know this error and can get a fix sometime this week.

**Comment by kee** [comment]
You are a god! Thank you.Kee NetheryOn Sep 19, 2023, at 6:51 PM, Elias Santistevan ***@***.***> wrote:﻿
Yes, I know this error and can get a fix sometime this week.

**Comment by edspark** [comment]
Error is solved in the latest release, thanks for the issue!

**Comment by kee** [comment]
And my project compiled with no errors. Thank you so much!
Kee
